src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -960,6 +988,21 @@ bool HMICapabilitiesImpl::load_capabilities_from_file() {
             soft_button_capabilities, soft_button_capabilities_so);
         set_soft_button_capabilities(soft_button_capabilities_so);
       }
+      if(check_existing_json_member(ui, "systemCapabilities")) {
+        Json::Value system_capabilities = ui.get("systemCapabilities", "");
+        if (check_existing_json_member(system_capabilities, "navigationCapability")) {
+          Json::Value navigation_capability = system_capabilities.get("navigationCapability","");
+          smart_objects::SmartObject navigation_capability_so;
+          Formatters::CFormatterJsonBase::jsonValueToObj(navigation_capability, navigation_capability_so);
+          set_navigation_capability(navigation_capability_so);
+        }
+        if (check_existing_json_member(system_capabilities, "phoneCapability")) {
+          Json::Value phone_capability = system_capabilities.get("phoneCapability","");
+          smart_objects::SmartObject phone_capability_so;
+          Formatters::CFormatterJsonBase::jsonValueToObj(phone_capability, phone_capability_so);
+          set_phone_capability(phone_capability_so);
+        }
+      }
```

Comment:
it could be added but none of the other hmi capabilities fields are validated at this time